.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -29,6 +28,13 @@ body:
       placeholder: "Explain what you expected to happen."
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: Actual Behavior
```

Comment:
I used to use `Steps to Reproduce` to show actual behaviour.
Maybe we don’t need the section `Steps to Reproduce` now?